src/renderers/dom/client/wrappers/__tests__/ReactDOMTextarea-test.js

```diff
@@ -178,27 +223,54 @@ describe('ReactDOMTextarea', function() {
 
     var container = document.createElement('div');
     var stub = <textarea>giraffe</textarea>;
-    stub = renderTextarea(stub, container);
-    var node = ReactDOM.findDOMNode(stub);
+    var node = renderTextarea(stub, container, true);
 
     expect(console.error.argsForCall.length).toBe(1);
     expect(node.value).toBe('giraffe');
 
-    // Changing children should do nothing, it functions like `defaultValue`.
+    // Changing children should cause value to change (new behavior of `defaultValue`)
     stub = ReactDOM.render(<textarea>gorilla</textarea>, container);
-    expect(node.value).toEqual('giraffe');
+    expect(node.value).toEqual('gorilla');
```

Comment:
Don’t we want to [remove support for `<textarea>{children}</textarea>` anyway](https://github.com/facebook/react/blob/master/src/renderers/dom/client/wrappers/ReactDOMTextarea.js#L114-L123). Might as well do that together with this. (Obv in 16 earliest.)